airflow/models/taskinstance.py

```diff
@@ -501,6 +524,13 @@ def refresh_from_task(self, task, pool_override=None):
         self.max_tries = task.retries
         self.executor_config = task.executor_config
         self.operator = task.__class__.__name__
+        # Mocks tags attribute is not iterable
```

Comment:
Don't put support code for tests in the real class -- provide a better mock instead.

Reply:
Fixed in fb4f595